containers/java/.devcontainer/base.Dockerfile

```diff
@@ -0,0 +1,46 @@
+# Update the VARIANT arg in devcontainer.json to pick a Java version: 11, 14
+ARG VARIANT=11
+FROM openjdk:${VARIANT}-jdk-buster
+
+# Options for setup script
+ARG INSTALL_ZSH="true"
+ARG UPGRADE_PACKAGES="false"
+ARG USERNAME=vscode
+ARG USER_UID=1000
+ARG USER_GID=$USER_UID
+
+# Install needed packages and setup non-root user. Use a separate RUN statement to add your own dependencies.
+COPY library-scripts/*.sh /tmp/library-scripts/
```

Comment:
Only copy `common-debian.sh`?

Reply:
Ack - looks like a missed a couple with a refactor.

Each section should copy its own script so that updating it will only rebuild that section.  If you copy all up front, any change to any script is a full rebuild.  Will fix - good catch!